build.sbt

```diff
@@ -44,6 +46,7 @@ lazy val s3 = project
     name := "async-blobstore-s3",
     libraryDependencies ++= Seq(
       "co.fs2" %% "fs2-core" % Fs2Version,
+      "io.netty" % "netty-all" % NettyVersion,
```

Comment:
Is netty-all the right thing to use here? Isn't that going to cause classpath conflicts with other artifacts which use a la carte netty dependencies (such as sdbe)?

Reply:
I think the problem is the other way around. Quasar already uses netty-all, see https://github.com/slamdata/quasar/blob/master/build.sbt#L290

Reply:
See also my comment in [ch10860]

Reply:
More clearly, I think a la carte is what can break things if the versions are not all the same, because this can lead to different versions being used at runtime. 
If we use netty-all consistently then I think we are fine, even if the patch part of the version does not match everywhere. Anyway, that's from what I have seen in my tests.

I didn't see any a la carte dep being specified in sdbe, but if we do somewhere I'd recommend using netty-all there too.